OnlineStoreTCA/CartList/Test/CartListDomainTest.swift

```diff
@@ -38,8 +38,7 @@ class CartListDomainTest: XCTestCase {
         
         let store = TestStore(
             initialState: CartListDomain.State(cartItems: cartItems),
-            reducer: CartListDomain.reducer,
-            environment: CartListDomain.Environment(
+            reducer: CartListDomain(
```

Comment:
I would like to refactor all tests to use Dependencies framework

Reply:
I created a new branch for dependencies in my fork from this branch.
You can have a look if you want that to be included in this MR:
**https://github.com/yashvyas29/OnlineStoreTCA/tree/feature/migration_to_dependency_management**
Functional part is working perfectly and I changed unit test part also but sometimes udid dependency bahaving randomly in unit tests showing error as unimplemented randomly. You can check that and let me know.